presto-main/src/main/java/com/facebook/presto/tracing/TracingException.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tracing;
+
+import com.facebook.presto.spi.PrestoException;
+
+import static com.facebook.presto.spi.StandardErrorCode.GENERIC_INTERNAL_ERROR;
+
+public class TracingException
+        extends PrestoException
+{
+    public TracingException(String message)
+    {
+        super(GENERIC_INTERNAL_ERROR, message);
```

Comment:
Let's add new tracing error code to standard error code instead of using GENERIC_INTERNAL_ERROR here. Also, we don't need to create `TracingException` class